style.css

```diff
@@ -0,0 +1,89 @@
+.toolbar {
+  height: 100px;
+  background: #fff;
+  box-shadow: 0 1px 0 #dddfe6;
+  display: flex;
+  align-content: center;
+  justify-content: center;
+  flex-wrap: wrap;
+}
+
+.nav {
+  width: 109px;
+  height: 24px;
+  font-family: 'Poppins', sans-serif;
+  font-style: normal;
+  font-weight: 400;
+  font-size: 16px;
+  line-height: 24px;
+  text-align: right;
+  color: #1a2236;
+}
+
```

Comment:
- [ ]  Kyndly follow the figma design as much as you can, as you can see in the mobile version of figma there is a background image on the top left corner, plese add it to your project. Take into account that if an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.

### Figma design
![image](https://user-images.githubusercontent.com/95243996/198582465-b71593b6-eaeb-47fc-8d81-d179cab70d75.png)

### Your project 

![image](https://user-images.githubusercontent.com/95243996/198582833-ac5e492a-6535-4019-add0-be17f5d23396.png)